integration/experiment/ffnet/README.md

```diff
@@ -0,0 +1,243 @@
+                   Frequency steering will occur at the package scope. If the path
+                   does not exist, an error will be thrown. At least one of 
+                   cpu-nn-path and gpu-nn-path must be specified or an error will be
+                   thrown.
+  - `gpu-nn-path`: A path to a neural network json for guiding GPU frequency decisions.
+                   Frequency steering will occur at the gpu scope. If the path
+                   does not exist, an error will be thrown. At least one of 
+                   cpu-nn-path and gpu-nn-path must be specified or an error will be
+                   thrown.
+  - `cpu-fmap-path`: A path to the CPU frequency recommendation map json. This file
+                     must contain region keys specified in the file at cpu-nn-path
+                     and must map each of these regions to an array of frequency 
+                     decisions for different values of perf-energy-bias. If cpu-nn-path
+                     is specified but cpu-fmap-path is not (or its path is invalid),
+                     an error will be thrown.
+  - `gpu-fmap-path`: A path to the GPU frequency recommendation map json. This file
+                     must contain region keys specified in the file at gpu-nn-path
+                     and must map each of these regions to an array of frequency 
+                     decisions for different values of perf-energy-bias. If gpu-nn-path
+                     is specified but gpu-fmap-path is not (or its path is invalid),
+                     an error will be thrown.
+
+## Additional Experiment Module
+#### `neural_net_sweep.py`:
+
+  Contains the helper function for launching a frequency sweep experiment, the data from
+  which can be used to generate neural net and region parameter json files. This function
+  executes a CPU/GPU/uncore frequency sweep while gathering telemetry that is useful to 
+  differentiate regions of interest.
+
+  In addition to command line arguments common to all runscripts, this script also accepts
+  options used by the `gpu_frequency_sweep experiment`:
+
+  **CPU Frequency Settings**
+
+  - `--max-frequency`: the maximum CPU frequency setting for the sweep.
+                       By default, it uses the sticker for the compute
+                       nodes, and executes an additional run with max
+                       turbo frequency as the limit.  Within each
+                       trial, the experiment script will start at the
+                       highest frequency and run at each limit until
+                       it reaches the minimum.
+
+  - `--min-frequency`: the minimum CPU frequency setting for the sweep.
+
+  - `--step-frequency`: the step size in hertz between CPU frequency
+                        settings for the sweep.
+ 
+  - `--run-max-turbo`: executes additional runs with max turbo
+                       frequency as the limit for the core frequency.
```

Comment:
Looking at the code, I see this parameter is optional. However, the documentation for ``--max-frequency`` makes it seem like a run with max turbo is always done. Is that the case? Some clarification might be needed. Also, I would recommend documenting the default behavior (max turbo = False).